content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+    <button class="btn btn-invisible ml-2">Ok, dismiss</button>
+  </div>
+</div>
+```
+
+- Show one teaching bubble at a time.
+- Recommended to include a headline that states the purpose of the message.
+- Keep your message short and pithy, around 160 characters. 
+- If additional information must be added, consider a stepped guide or a “Learn more” link that opens in a new window or tab. 
+- It is preferable to use a descriptive dismiss button (e.g. “Ok, got it” or “Done”) over a close icon; if a close icon must be used, remember to apply the correct tab order and polite annoucements for screenreaders.
+-If a guided feature setup is needed in a recurring manner, consider using other more inline  patterns like empty states in lieu of teaching bubbles.
+- Keep your copy to maximum 160 characters.
+
+### Multiple calls to action
+The default dismissal button should say “Ok, got it” as a way to inject more personality while reaffirming with “ok” which internationalizes better as a more commonly recognized phrase. As a secondary call to action, it should be “Ok, dismiss.”
+
+<p><img src="https://user-images.githubusercontent.com/6744014/88741737-43b93e80-d0f5-11ea-8445-5b878607756e.png" width="180"/></p>
+
+If the primary action is not to dismiss the teaching bubble, they should invite the user to engage with the value proposition promised. Call to actions in teach bubbles should not take the user away from their current page. 
+
+Actions that take the user to a new page may only be linked from the copy and must open in a new tab or window to prevent loosing the user’s context.
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={9 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/6744014/88740809-f340e180-d0f2-11ea-9ba3-15005aebc378.png" />
+  </BorderBox>
+  <p>Multiple call to actions</p>
+</Flex>
+
+### Variations
+
+|Emoji|Pictograms|Illustrations and gifs|
+|---|---|---|
+|![emoji](https://user-images.githubusercontent.com/6744014/88740799-f045f100-d0f2-11ea-9d23-5d002b2510e3.png)|![packages](https://user-images.githubusercontent.com/6744014/88740806-f2a84b00-d0f2-11ea-9530-f144d47e331c.png)|![gif](https://user-images.githubusercontent.com/6744014/88740801-f0de8780-d0f2-11ea-8c0f-1a52e2f38da0.png)|
+
+**Condensed**
+
+Use a close icon only for condensed messages that have no primary call to action. Remember to set the correct aria order for screen readers to easily dismiss the teaching bubble at the end of the message.
+
+<p><img src="https://user-images.githubusercontent.com/6744014/88740797-efad5a80-d0f2-11ea-95dc-8781f046f70c.png" width="300"/></p>
+
+# Inline feature discovery
+
+A banner that appears inline often used as a way to reveal hidden features in more obscure parts of GitHub. Do not persist the banner without an ability to dismiss on a “core” high traffic page.
```

Comment:
Can we remove the double negative?

Reply:
Just noticing, the first sentence is missing a start, like "Inline feature discovery is..."